# Skills
Skill development: from beginner to proficiency

[Qing Li CV](https://www.overleaf.com/project/68790edeb8f258d92232e63a)

## TECHNICAL SKILLS
â€¢ Programming: Python, R, Bash (Linux, HPC, slurm), SQL, SAS
â€¢ Multi-omics deep learning models: DNABERT, Enformer, Boltz-2 (genomic); iSTART, iSCALE,
Squidiff (single-cell/spatial); vision transformer, FastSurfer, EchoPrime (imaging)
â€¢ Spatial Transcriptomics: 10X Visium, NanoString GeoMx
â€¢ Electronic Health Records (EHR) Analysis: IPTW framework, Cox proportional hazards modeling for
comparative drug effectiveness
â€¢ Next-generation sequencing (NGS), Germline/Somatic variant detection(GATK), Variants
interpretation (Ensembl VEP), bulk and single-cell RNA sequencing, proteomics
â€¢ Genome-wide Association Studies (GWAS), Transcriptome-wide Association Studies (TWAS),
GWAS/TWAS Fine Mapping
â€¢ Causal Inference: coloc, Mendelian Randomization (MR)
â€¢ Other tools: Power BI, microsoft office, AWS, GitHub, Nextflow [Cheet sheet](Documents/nextflow_cheatsheet.pdf)

### Single-cell analysis
- **scTDA**: Topological data analysis of high-throughput single-cell RNA-seq data, applied to studying cell differentiation during development.  
  - ðŸ“„ PMID: [28459448](https://pubmed.ncbi.nlm.nih.gov/28459448/)  
  - ðŸ“˜ Tutorial: [scTDA Tutorial](Documents/scTDA Tutorial.ipynb)